spec/settings_spec.rb

```diff
@@ -41,7 +42,6 @@ class TestEngine < ::Rails::Engine
   it "saves to db and returns persisted values" do
     expect(TestEngine::settings.host).to eq('localhost')
     TestEngine::settings.host = '127.0.0.1'
-    expect_any_instance_of(TestEngine::settings::send(:configuration_class)).to receive(:update_attribute)
```

Comment:
configuration_class method was removed [here](https://github.com/dradis/dradis-plugins/pull/2/files#diff-5773e9b94584306186f2e4ad96db8100d666a97f15b78ca3c464b928760cafca) seems like this test is not needed anymore.